src/pybind/mgr/dashboard/controllers/mgr_modules.py

```diff
@@ -114,16 +141,28 @@ def _get_module_options(self, module_name):
         :rtype: dict
         """
         options = self._get_module_config(module_name)['module_options']
-        # Workaround a possible bug in the Ceph Mgr implementation. The
-        # 'default_value' field is always returned as a string.
+        return self._convert_module_options(options)
+
+    def _convert_module_options(self, options):
+        # Workaround a possible bug in the Ceph Mgr implementation.
```

Comment:
> I was wondering whether it would make sense to set defaults as initial values during `PyModule::load_options()` if no value has been ever set, and offload the mgr_module of the default handling logic.

I was also thinking about that, but i am not sure if there is a special reason why this feature is actually missing. Since setting a default value in the processing is actually something fundamental, I assume that the original author had compelling reasons not to do so.